content/docs/user-guide/platforms/android/_index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Android Support for Open 3D Engine (O3DE)
+linkTitle: Android
+description: An overview of Open 3D Engine support for development for Android.
+weight: 200
+---
+
+O3DE projects for Android relies on the gradle build system to build the final APK required to launch and run O3DE projects. The gradle build scripts themselves uses cmake as a subset of the build process to compile the native C++ code, while gradle will also handle the build of the android application (java) and construction and package creation of the final APK and assets. 
+
+Because of this android projects require a custom script to generate the project beyond what cmake does by itself.
+
+## **Prerequisite Software and Packages**
+
+### **Java Development Kit (JDK)**
+The latest version of the Java Development Kit is needed to both compile the android java source for the project as well as to create a Java Key Storage (JKS) files used by the Android SDK to sign the APK.
```

Comment:
I'm not sure, i've been hesitant about linking download specific page URLs, rather opting to use the main URL in case they change. Maybe we can just use the main java page (https://www.oracle.com/java/) instead and allow them to search for the JDK specifically

Reply:
That's true, it's common for specific URLs to change, making information in this doc obsolete quickly. I agree that we should use https://www.oracle.com/java/ instead.